gradle/dependencies.gradle

```diff
@@ -7,7 +7,9 @@ dependencies {
     testImplementation 'io.opentelemetry:opentelemetry-api:1.37.0'
     testImplementation 'io.opentelemetry:opentelemetry-context:1.37.0'
     testImplementation 'io.github.std-uritemplate:std-uritemplate:0.0.56'
-    
+    testImplementation 'io.opentelemetry.semconv:opentelemetry-semconv:1.25.0-alpha'
+    testImplementation 'io.opentelemetry.semconv:opentelemetry-semconv-incubating:1.25.0-alpha'
+
     implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
we should also update the kiota dependencies, or wait for dependabot to do the work.